includes/class-wc-stripe-payment-information.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Class Payment_Information
+ *
+ * @package WooCommerce\Stripe
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Mostly a wrapper containing information on a single payment.
+ */
+class WC_Stripe_Payment_Information {
```

Comment:
Do we need this class at all? Since there's only one function in it and no other properties included in the class, could this not just be added as a function inside the payment gateway?

Reply:
Good point; this class is only used by the save payment functionality (and is not part of this PR, will be added after #1711). I'll remove this for now.